constants.py

```diff
@@ -1,12 +1,40 @@
 import os
 import numpy as np
 
-# Things anyone wishing to use this repository for their
-# own use will want to change
-MEDIA_DIR = os.path.join(
-    os.path.expanduser('~'),
-    "Dropbox (3Blue1Brown)/3Blue1Brown Team Folder"
-)
+env_MEDIA_DIR = None
+MEDIA_DIR = "#ERROR#"
+
+try:
+    env_MEDIA_DIR = os.getenv("MEDIA_DIR")
+except NameError:
+    try:
+        env_MEDIA_DIR = os.environ['MEDIA_DIR']
+    except KeyError:
+        pass
```

Comment:
There are 2 ways you try getting env. Is this legitimate way to get it?

Reply:
I think it's best to allow multiple possibilities to get an environment variable. (Note: `os.getenv(...)` is Windows/Unix only and `os.environ[...]` *may* be unstable.)